src/Interface/User.tsx

```diff
@@ -0,0 +1,12 @@
+import { Ticket } from "../Interface/Ticket";
+
+export interface User {
```

Comment:
Where is user role here?

Reply:
Should I implement user role and if yes, what do you have in mind for that implementation?

Reply:
This from `NavigationBar.tsx`:
```typescript
export enum UserRole {
    Super,
    Admin,
    User,
    none
}
```

Reply:
Normally, the change is done by now.